### PR TITLE
Add command history widget

### PR DESCRIPTION
**Pull request by ceski-1**
This is typically used for speedrunning but I also find it useful for debugging. Adapted from DSDA-Doom.

Fixes https://github.com/fabiangreffrath/woof/issues/1284

### CODE REVIEW

File: autoload/all-all/woofhud.lmp [hunk 1]
```diff
@@ -4,6 +4,7 @@ monsec topleft
 sttime topleft
 coord topright
 fps topright
+cmd bottomright
```
Comment (ceski-1) [inline autoload/all-all/woofhud.lmp:7]: Currently, multi-line widgets don't work properly when using absolute screen coordinates.

File: src/mn_setup.c [hunk 2]
```diff
@@ -1438,6 +1438,9 @@ static setup_menu_t stat_settings2[] = {
 
     {"Use-Button Timer", S_ONOFF, M_X, M_SPC, {"hud_time_use"}},
 
+    {"Command Display", S_ONOFF | S_STRICT, M_X, M_SPC, {"hud_command_display"},
```
Comment (fabiangreffrath) [inline src/mn_setup.c:1441]: How about "Input History"?

File: autoload/all-all/woofhud.lmp (hunk 1 above)
Comment (fabiangreffrath) [reply autoload/all-all/woofhud.lmp:7]: No, unfortunately not.

File: src/mn_setup.c (hunk 2 above)
Comment (ceski-1) [reply src/mn_setup.c:1441]: "Command Display" has been the established name for a couple years now, but something like "Show Input Commands" or your suggestion is probably more intuitive.

Comment (fabiangreffrath) [reply src/mn_setup.c:1441]: Yes, please.

Comment (ceski-1) [reply src/mn_setup.c]: Let's go with "Show Command History". Fixed by https://github.com/fabiangreffrath/woof/pull/1692/commits/52d1689d1833160ae80bddbe1d4fe44c1d7407a4

File: autoload/all-all/woofhud.lmp (hunk 1 above)
Comment (ceski-1) [reply autoload/all-all/woofhud.lmp:7]: Should we consider this out of scope for this PR?

Comment (fabiangreffrath) [reply autoload/all-all/woofhud.lmp:7]: I agree it should be fixed, but this is orthogonal to the work here.

**Comment by ceski-1** [comment]
Are there any other comments? Is it okay to merge this?

**Review by fabiangreffrath — APPROVED** [review]
Looks good code-wise, but I hadn't had a chance to check it out and actually run it. I trust you that it works as announced so feel free to merge.

**Review by rfomin — APPROVED** [review]
Works good for me, thanks.

**Pull request merged**